docker/postgres-initdb/ichiran-db.sh

```diff
@@ -5,6 +5,7 @@ echo "========================="
 createdb -E 'UTF8' -l 'ja_JP.utf8' -T template0 dummy
 set +e
 pg_restore -C -d dummy /ichiran.pgdump --no-owner --no-privileges
+psql -c "ALTER DATABASE $ICHIRAN_DB_NAME_IN_URL RENAME TO jmdict"
```

Comment:
Wait so does -C actually ignore the specified database name and create a database with the original name? I didn't know that. Maybe `-C` option is unnecessary and it's only needed to just rename `dummy` to `jmdict`.

Reply:
Yup you're 100% right, thanks for noticing that! Yes this `-C` is the root of all my problems here. Removing it and just naming the new database correctly from the start is a much simpler approach!

Oops, out of habit I force-pushed my branch, hopefully that's ok 🙇